# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/jsaw/scribus/math.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


The usual pow, ... builtin for x86 - but not for many others.

  - Rene Rebe <rene@exactcode.de>

--- ./scribus/scribusview.cpp.vanilla	2004-06-12 15:23:03.058045128 +0200
+++ ./scribus/scribusview.cpp	2004-06-12 15:23:18.707666024 +0200
@@ -25,6 +25,7 @@
 #endif
 
 #include "pageback.h"
+#include <math.h>
 #include <qcolor.h>
 #include <qfont.h>
 #include <qfontmetrics.h>
--- ./scribus/scribus.cpp.vanilla	2004-06-12 15:25:20.734115184 +0200
+++ ./scribus/scribus.cpp	2004-06-12 15:25:47.676019392 +0200
@@ -29,6 +29,8 @@
 #include <qvbox.h>
 #include <cstdio>
 #include <cstdlib>
+#include <math.h>
+
 #include <dlfcn.h>
 #include <unistd.h>
 #include <iostream>
